jans-cedarling/cedarling/src/init/policy_store.rs

```diff
@@ -0,0 +1,66 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use crate::models::policy_store::{PolicyStore, PolicyStoreMap};
+use crate::models::policy_store_config::{PolicyStoreConfig, PolicyStoreSource};
+
+/// Error cases for loading policy
+#[derive(Debug, thiserror::Error)]
+pub enum ErrorLoadPolicyStore {
```

Comment:
`PolicyStoreError`

Reply:
renamed to the `LoadPolicyStoreError`
because this error related to the function `load_policy_store_map`
and enum with source cases will be expanded